tests/foreman/cli/test_host.py

```diff
@@ -2051,7 +2053,7 @@ def test_negative_multi_cv_registration(
 
     :CaseImportance: Critical
 
-    :parametrized: yes
+    :parametrized: no
```

Comment:
`no` is correct here since my tests don't have `@pytest.mark.parametrize`, right? I'm still new to all these fields. I'm hoping reviewers will tell me if they are wrong.

Reply:
When we don't specify it, the default is no, so you can omit this line in test.

Reply:
On the other hand, the test is parametrized by the `rhel_ver_match` marker. I believe that we have no strict guidelines for this.

Reply:
@jeremylenz I'd vote for `yes` here for consistency with other tests, when you run the module, pytest will spawn a bunch of tests based on `@pytest.mark.rhel_ver_match('[^7]')`, iirc that `parametrized` docstring was needed for polarion, not sure if that is still a thing these days

Reply:
I'd vote `yes` as well from prior comments, since you are parameterizing `rhel_contenthost` by rhel7, 8, and 9 only (3 cases).
Non-parameterized without `rhel_ver_match`, would give you 6 cases. 
One for each rhel7 to 9, and rhel7 to 9-fips